addons/rose/tests/dummy/app/templates/index.hbs

```diff
@@ -1,26 +1,31 @@
 <Rose::Toolbar as |toolbar|>
-  <Rose::Button
-    @style='inline-link-action'
-    @iconLeft='flight-icons/svg/x-16'
-  >Clear somthing</Rose::Button>
+  <Rose::Button @style='inline-link-action' @iconLeft='flight-icons/svg/x-16'>
```

Comment:
To keep consitency with lines below, Do you think we can move the `@style` and `@iconLeft` to new nested lines?

Reply:
I had used prettier to format it and I think it didn't move them into newlines because it didn't go past the character length limit we have of 80 (it was at 79). 

The other ones have whitespace for indentation which pushed them past 80 so it nested them. 
It is a little weird that it doesn't look consistent but that's what the formatter does 🤷